.github/workflows/container.yml

```diff
@@ -6,13 +6,18 @@ on:
       - 'master'
       - 'container'
     paths:
+      - '.github/workflows/container.yml'
       - 'test/Dockerfile'
   workflow_dispatch: {}
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+    - name: Install QEMU
```

Comment:
This seems to be unrelated to `build on workflow changes`, thus maybe put it in another commit.

Reply:
Move to `.github/workflows/container: build multi-platform image`.